### DISCUSSION TITLE
Questions about solvent miscibility

### DISCUSSION THREAD

**Discussion by rswf** [ricalessandri/Martini3-small-molecules#54] [Q&A] [answered]
Hello! @ricalessandri 
I am currently working on simulations of polymer behavior in mixed solvents involving a range of solvents such as ethanol, styrene, toluene, dioxane, cyclohexane, and others. Thank you very much for your work based on Martini3 force field small molecule parameterization, which is very helpful for me.

However, testing revealed that the two models proposed in your work, toluene and styrene, are not very miscible with ethanol. Most of the toluene and styrene molecules are agglomerated rather than uniformly distributed in ethanol, which is not consistent with reality. 

The volume fraction of both toluene and styrene was 12%. Simulations were performed at 300 K, 1 bar. Results were obtained by starting the simulation from a randomly distributed initial configuration.

Toluene, styrene and ethanol are carefully parameterized to get the model, and it may not be more accurate to reconstruct the model myself. Would it make sense to use the additional labels provided by Martini to adjust the interactions within the solvent model? Or do you have any suggestions?

![image](https://github.com/user-attachments/assets/b27e9f0b-55e8-41aa-90a6-512ea2fd3b7d)
EOL-TL(12%)

![image](https://github.com/user-attachments/assets/284f08e3-8e95-4611-a009-f8e7e48cfcdb)
EOL-STYR(12%)

**Comment by ricalessandri** [comment, marked as answer]
@rswf thanks for reaching out. I'd expect these mixtures to be problematic. If you look at Supp. Table 27 of the [Martini 3 paper](https://www.nature.com/articles/s41592-021-01098-3):

<img width="700" alt="Screen Shot 2024-07-13 at 6 53 19 AM" src="https://github.com/user-attachments/assets/1d3c472d-b17a-4aa0-acac-7b444a0dc0a5">

you can see that among the mixtures that M3 does not get right (the ones with an "X"), there's ethanol-benzene and ethanol-xylene. The mixtures of your interest are practically the same. As we write when discussing Supp Table 27:
_"Overall, Martini 3 predicted the correct behavior in 86% of the cases. Currently, the most problematic solvents are acetic acid, carbon-tetrachloride, methanol and ethanol, which represented 50% of the wrong phase assignments. They usually tend to phase separate too much from other solvents. Further investigations are necessary to understand if the current problems are related with bead assignment of such compounds or if the pair interactions are involved."_ So, the problem with ethanol is exactly what you encounter (too much phase separation). Since then, I haven't done any further tests myself - and I'm not aware of anybody else that did it - on ethanol.

What you suggest - trying labels - is a good starting point. I would focus on changing ethanol, as toluene and styrene have been validated more extensively. Labels would be the "least invasive" way to update the ethanol model. However, as we write in that paragraph, you could also try different bead assignments. Feel free to follow up here if you want further feedback after some tests.

**Reply by rswf** [reply]
@ricalessandri Thank you for your answer, it clears up my confusion. I will try to tweak the ethanol model as you suggested.
